### PR TITLE
Add snippet of where the issue occurs

### PR DESCRIPTION
**Pull request by omrilotan**
Test this branch:

```
npm i ecma-webpack-plugin@rc -D
```

### CODE REVIEW

File: lib/errors/index.js [hunk 1]
```diff
@@ -19,8 +19,34 @@ module.exports = (
...
 				parse(content, parser);
-			} catch ({ message }) {
-				accumulator.push({ name, message });
+			} catch (error) {
+				const { message, loc: { line, column } = {}, raisedAt } = error;
+				const issue = { name, message };
+				if (line && column) {
+					const lineOfCode = (content || '').split('\n')[line - 1];
+
+					const [ prefix, startFrom, indentation ] = column > 20
```
Comment (sahariko) [inline lib/errors/index.js]: Maybe worth assigning this to a meaningful variable, like `shouldTrimStart`.

Same for trimming the end.

Comment (omrilotan) [reply lib/errors/index.js]: Just for naming the condition, do you think?

Comment (sahariko) [reply lib/errors/index.js]: I had a little bit of a hard time what this check is for, and what roles play `20` and `30`.

Comment (omrilotan) [reply lib/errors/index.js]: Sounds good. I've added some named variables to make this more readable

**Pull request merged**